NuGet.config

```diff
@@ -18,6 +18,7 @@
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
```

Comment:
We're actively working on moving away from all uses of myget, so this would block that effort. Can we uploaded the packages elsewhere?

FYI to @dibarbet.

Reply:
Please don't add a myget source. I am hoping we can get the M.CA.Testing packages published so that this can merge today https://github.com/dotnet/roslyn/pull/46220

Reply:
Is this a feed that we've created or one that xunit has created for themselves? Assuming the latter. 

I don't think we're as concerned about the latter for the myget removal effort. That is mostly focused on removing `dotnet.myget.org` references. But at the same time the simpler message of "we're not using myget" is simpler and would be nice if we could keep it.

Reply:
@jaredpar is correct.  We're about to delete/remove dotnet.myget.org, and our general guidance is to not use myget at all.  

Is there a way to help them move?  (I'm not even sure "who" xunit is these days)  We'd (eng svcs) be happy to help w/ guidance etc....

Reply:
➡️ Now pulls from dotnet-eng feed